README.md

```diff
@@ -348,14 +376,46 @@ renderChat(
 >
 > If browser disable or cannot handle locale storage, the chat will not store messages.
 
-#### Opening message
+#### Post-initialization behaviours
+
+##### After Init callback
+
+The optional `afterInit` is a callback function, called once the chat is ready to send payloads to the server.
+It takes a `UseTock` parameter, which allows the callback to interact with the chat (sending messages, clearing the chat, etc.).
+If it returns a promise, the chat will wait for the promise to resolve before performing other post-init tasks.
+
+Example:
+
+```js
+renderChat(
+    document.getElementById('chat'), 
+    '<TOCK_BOT_API_URL>', 
+    undefined,
+    {},
+    {
+      localStorageHistory: { enable: true },
+      afterInit: async (tock) => {
+        if (tock.messages.length) await tock.sendMessage("I'm back");
+        tock.addImage('Nice boat', '/nice-boat.png');
+      }
+    },
+);
+```
+
+In this example, whenever the page is loaded, the following sequence will play:
+- If the user previously interacted with the bot, they will send the "I am back message" to the bot endpoint,
+triggering the corresponding story.
+- In any case an image will be displayed as a bot message. The latter message is clientside only and will not be recorded to the history.
+
+##### Opening message
 
 The optional `openingMessage` is a sentence, automatically sent to the bot when the conversation starts.
-This is typically used to provide a welcoming or onboarding message to the user:
-- configured in _Tock Studio_ or managed like any other Tock story,
-- not requiring the user to make a sentence first.
+If an `afterInit` callback is also specified, it runs before the opening message gets sent.
+This is typically used to trigger a welcoming or onboarding story for the user,
+without requiring them to type a sentence first.
+Said story is configured in _Tock Studio_, or managed like any other Tock story,
 
-The `openingMessage` parameter is a sentence from the user to the bot, actually not displayed in conversation.
+The `openingMessage` parameter is a sentence from the user to the bot, albeit not actually displayed in conversation.
```

Comment:
albeit ?

Reply:
Thought it would be the most appropriate word in this sentence (cf. https://www.merriam-webster.com/dictionary/albeit), would you suggest something else ?